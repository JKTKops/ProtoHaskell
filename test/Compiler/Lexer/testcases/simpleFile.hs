module SimpleFile where

foo :: Foo
foo = undefined

bar :: (Bar -> Foo) -> Bar -> Foo
bar f b = f b

bar' :: (Bar -> Foo) -> Bar -> Foo
bar' = id
