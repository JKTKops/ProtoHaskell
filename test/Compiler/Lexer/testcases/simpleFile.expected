Right [<indentation>, module, SimpleFile, where, <indentation>,
       foo, ::, Foo, <indentation>, foo, =, undefined, <indentation>, bar,
       ::, (, Bar, ->, Foo, ), ->, Bar, ->, Foo, <indentation>, bar, f, b,
       =, f, b, <indentation>, bar', ::, (, Bar, ->, Foo, ), ->, Bar, ->,
       Foo, <indentation>, bar', =, id]